exercises/01-composite-ui/before/01-composite-ui-before.sln

```diff
@@ -129,6 +128,7 @@ Global
 		{5BC1FA4F-BA4F-46F0-836F-D457F5CE2428} = {3F113598-B240-4E06-B41A-6C684C208F2A}
 		{63B7E9CB-C735-4FC3-B139-9280F9F102E5} = {808B901F-6EF7-49F0-B668-F54EEDB1F3F1}
 		{C0FEE1F1-7540-4340-A05D-B776032584C3} = {65F37558-1C29-4247-BD01-4DFAFD452ECB}
+		{FDD10539-A381-48BD-AA56-328DD9991B9F} = {808B901F-6EF7-49F0-B668-F54EEDB1F3F1}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {FC27BA8B-135F-4456-92CC-AE23DD9989AB}
```

Comment:
Not so relevant right now, but it seems we should revert all changes to this file before merging.